teuthology/openstack/setup-openstack.sh

```diff
@@ -0,0 +1,548 @@
+#!/bin/bash
```

Comment:
I think you mentioned yesterday on the phone that you intend to (or intend to help) obsolete this with ansible or even just python; perhaps a comment at the top noting that would be helpful in case we both disappear :)